Configs/Secrets.swift.example

```diff
@@ -40,8 +40,8 @@ public enum Secrets {
   }
 
   public enum Segment {
-    public static let staging = "write-key"
-    public static let production = "write-key"
+    public static let staging = "write-key-production"
+    public static let production = "write-key-staging"
```

Comment:
looks like you mixed these up

Reply:
oh lol yeah, they mostly just need to not be the same but best have them the right way around 👍